openwisp_radius/tests/test_api/test_freeradius_api.py

```diff
@@ -677,28 +680,45 @@ def test_accounting_no_token_403(self):
         )
 
     @freeze_time(START_DATE)
-    def test_accounting_start_200(self):
+    @mock.patch('openwisp_radius.receivers.send_login_email')
```

Comment:
```suggestion
    @mock.patch('openwisp_radius.receivers.send_login_email.delay')
```

It is required to mock the exact method that is called.

Reply:
It still fails.

Reply:
In `openwisp_radius.apps`, `radius_accounting_success` is connected to `openwisp_radius.api.freeradius_views.AccountingView`

https://github.com/openwisp/openwisp-radius/blob/6facba5814022303c70ab88ed4a382979af5202f/openwisp_radius/apps.py#L69-L73

But the sample app inherits the base view and creates a new class: 

https://github.com/openwisp/openwisp-radius/blob/6facba5814022303c70ab88ed4a382979af5202f/tests/openwisp2/sample_radius/api/views.py#L32-L33

The sender of the signal is this new class for sample application. We need to connect this signal in apps.py of sample app.